x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/external/http/HttpSettings.java

```diff
@@ -20,9 +20,9 @@ public class HttpSettings {
     // These settings are default scope for testing
     static final Setting<ByteSizeValue> MAX_HTTP_RESPONSE_SIZE = Setting.byteSizeSetting(
         "xpack.inference.http.max_response_size",
-        new ByteSizeValue(10, ByteSizeUnit.MB),   // default
+        new ByteSizeValue(50, ByteSizeUnit.MB),   // default
         ByteSizeValue.ONE, // min
-        new ByteSizeValue(50, ByteSizeUnit.MB),   // max
+        new ByteSizeValue(100, ByteSizeUnit.MB),   // max
```

Comment:
Just a reminder to update the docs with these changes.

Reply:
Thanks for the reminder, I updated the docs